spec/generators/dashboard_generator_spec.rb

```diff
@@ -142,7 +142,7 @@ class Shipment < ActiveRecord::Base
           attrs = ShipmentDashboard::ATTRIBUTE_TYPES
 
           expect(attrs[:status]).
-            to eq(Administrate::Field::String.with_options(searchable: false))
+            to eq(Administrate::Field::Enum.with_options(enum: {'ready' => 0, 'processing' => 1, 'shipped' => 2}))
```

Comment:
Line is too long. [114/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.